crates/holochain_conductor_config/src/ports.rs

```diff
@@ -0,0 +1,50 @@
+//! Helpers for working with ports.
+
+use holochain_conductor_api::{
+    config::conductor::ConductorConfig, AdminInterfaceConfig, InterfaceDriver,
+};
+use holochain_types::websocket::AllowedOrigins;
+
+use crate::msg;
+
+pub fn random_admin_port(config: &mut ConductorConfig) {
```

Comment:
Without `set` it sounds more like a getter.
```suggestion
pub fn set_random_admin_port(config: &mut ConductorConfig) {
```